src/pages/worklist/DoiRequestAccordion.tsx

```diff
@@ -60,46 +63,51 @@ const StyledAccordionActionButtons = styled.div`
 `;
 
 interface DoiRequestAccordionProps {
-  registration: Registration;
+  identifier: string;
 }
 
-export const DoiRequestAccordion: FC<DoiRequestAccordionProps> = ({ registration }) => {
+export const DoiRequestAccordion = ({ identifier }: DoiRequestAccordionProps) => {
   const { t } = useTranslation('workLists');
-  const {
-    identifier,
-    owner,
-    entityDescription: { mainTitle },
-    doiRequest,
-  } = registration;
+  const dispatch = useDispatch();
+  const [isSendingMessage, setIsSendingMessage] = useState(false);
+  const [registration, , refetchRegistration] = useFetchRegistration(identifier);
 
-  if (!doiRequest) {
+  if (!registration?.doiRequest) {
     return null;
   }
 
+  const onClickSendMessage = async (message: string) => {
+    setIsSendingMessage(true);
+    const updatedDoiRequestWithMessage = await updateDoiRequestWithMessage(identifier, message);
+    if (updatedDoiRequestWithMessage) {
+      if (updatedDoiRequestWithMessage.error) {
+        dispatch(setNotification(t('feedback:error.update_doi_request'), NotificationVariant.Error));
+      } else {
+        dispatch(setNotification(t('feedback:success.doi_request_updated'), NotificationVariant.Success));
```

Comment:
kanskje legg inn en egen melding for "Melding sendt" elns? "DOI-forespørselen ble oppdatert" er kanskje litt kryptisk, og vi kan trenge det for support etc i fremtiden også

Reply:
ja, kan gjøre det 👍